os/linux/gbmbufferhandler.cpp

```diff
@@ -68,44 +68,43 @@ bool GbmBufferHandler::CreateBuffer(uint32_t w, uint32_t h, int format,
   if (gbm_format == 0)
     gbm_format = GBM_FORMAT_XRGB8888;
 
-  struct gbm_bo *bo = gbm_bo_create(device_, w, h, gbm_format,
-                                    GBM_BO_USE_SCANOUT | GBM_BO_USE_RENDERING);
+  gbm_bo_ = gbm_bo_create(device_, w, h, gbm_format,
```

Comment:
ao, maybe involve in double release/destory risk of bo if keep track bo on both handle and GbmBufferHandler